pulp_ansible/app/tasks/collectionversion_index.py

```diff
@@ -35,6 +37,40 @@ def get_highest_version_string_from_cv_objects(cv_objects):
     return str(versions[0])
 
 
+def compute_repository_changes(repository_version):
+    """Use the previous version to make a list of namespace(s).name(s) changed."""
+
+    # Figure out what the previous repo version is
+    repository = repository_version.repository
+    previous_number = repository_version.number - 1
+    previous_version = RepositoryVersion.objects.filter(
+        repository=repository, number=previous_number
+    ).first()
+
+    # If there isn't a previous verison, all things have "changed"
+    if previous_version is None:
+        return None
+
+    changed_collections = set()
+
+    for method in ["added", "removed"]:
+        func = getattr(repository_version, method)
+        for modified in func(base_version=previous_version):
+            if modified.pulp_type == CollectionVersion.get_pulp_type():
+                cv = modified.ansible_collectionversion
+                changed_collections.add((cv.namespace, cv.name))
+            elif modified.pulp_type == AnsibleCollectionDeprecated.get_pulp_type():
+                dep = modified.ansible_ansiblecollectiondeprecated
```

Comment:
I know, the scope is small, but "dep" is a bit ambiguous. (Sounds like dependency to me.)